### DISCUSSION TITLE
Neverball 1.7.0-alpha.1

### DISCUSSION THREAD

**Discussion by parasti** [Neverball/neverball#362] [Announcements]
Just a quick prerelease so I can upload a Windows build.

<hr /><em>This discussion was created from the release <a href='https://github.com/Neverball/neverball/releases/tag/1.7.0-alpha.1'>Neverball 1.7.0-alpha.1</a>.</em>

**Comment by qwertychouskie** [comment]
Hyyyyyyype

Though with the amount of improvements, maybe it's worth calling it 2.0?
